smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolServiceClient.kt

```diff
@@ -190,13 +195,15 @@ open class HttpProtocolServiceClient(
 
     private fun renderConfigInitializer(serviceSymbol: Symbol, properties: List<ConfigProperty>) {
         writer.openBlock("private init(", ") {") {
```

Comment:
In several places below, the `renderProperties` helper function was removed, since (1) using `writer.unwrite` is a better approach, and (2) it doesn't handle nested indent correctly.